plugin/build.gradle.kts

```diff
@@ -106,13 +106,17 @@ dependencies {
     "rewriteDependencies"("org.openrewrite:rewrite-protobuf")
     "rewriteDependencies"("org.openrewrite:rewrite-groovy")
     "rewriteDependencies"("org.openrewrite:rewrite-gradle")
+    "rewriteDependencies"("org.openrewrite.recipe:rewrite-all:$latest") {
+        isTransitive = false
+    }
     "rewriteDependencies"("org.openrewrite.gradle.tooling:model:$latest")
     "rewriteDependencies"("org.openrewrite:rewrite-maven")
     // Newer versions of checkstyle are compiled with a newer version of Java than is supported with gradle 4.x
     "rewriteDependencies"("com.puppycrawl.tools:checkstyle:9.3")
     "rewriteDependencies"("com.fasterxml.jackson.module:jackson-module-kotlin:latest.release")
 
     implementation(platform("org.openrewrite:rewrite-bom:$latest"))
+    implementation("org.openrewrite.recipe:rewrite-all:$latest")
```

Comment:
I think the use case for this is covered by the `compileOnly` and would otherwise be leaking the OpenRewrite libraries back onto the plugin classpath.